docs/elements.md

```diff
@@ -184,8 +184,8 @@ as checkboxes.
 | `max-correct`           | integer | special | The maximum number of correct answers to display. Defaults to displaying all correct answers.                                                                                                                                        |
 | `fixed-order`           | boolean | false   | Disable the randomization of answer order.                                                                                                                                                                                           |
 | `partial-credit`        | boolean | false   | Enable partial credit scores. By default, the choice of grading method is "all-or-nothing".                                                                                                                                          |
-| `partial-credit-method` | string  | 'PC'    | Three grading methods for partial credit: 'COV' (Coverage), 'EDC' (Every Decision Counts), and 'PC' (Percent Correct). See explanation below.                                                                                        |
-| `hide-help-text`        | boolean | false   | Help text with hint regarding the selection of answers. Popover button describes the selected grading algorithm ('all-or-nothing', 'COV', 'EDC' or 'PC')                                                                             |
+| `partial-credit-method` | string  | `"PC"`  | Three grading methods for partial credit: "COV" (Coverage), "EDC" (Every Decision Counts), and "PC" (Percent Correct). See explanation below.                                                                                        |
+| `hide-help-text`        | boolean | false   | Help text with hint regarding the selection of answers. Popover button describes the selected grading algorithm ("all-or-nothing", "COV", "EDC" or "PC")                                                                             |
```

Comment:
@eliotwrobson totally bikeshedding here: do you think we should backtick enum values when they're used in sentences, like the list of grading algorithms here? Within this PR, we're not consistent between elements; would be nice to decide on a consistent style while we're here.

Reply:
Just to confirm, do we want to format values in the description column as well or is this just limited to the type and default columns?

Reply:
I think we do also want to format values with backticks in the description columns (and elsewhere too).